src/pages/studyView/addChartButton/AddChartButton.tsx

```diff
@@ -292,14 +292,24 @@ class AddChartTabs extends React.Component<IAddChartTabsProps, {}> {
             charts,
             this.selectedAttrs
         );
-        // TODO: (GA) Add other datatype by using another function
         if (
             _.every(
                 charts,
                 chart => chart.dataType === DataTypeConstants.LIMITVALUE
             )
         ) {
             this.props.store.addGenericAssayContinuousCharts(charts);
+        } else if (
```

Comment:
this whole if statements seems slightly strange and needs comment.  seems like why do all charts need to be of the same type in order to submit?  i mean, i would assume charts are independent of each other.

Reply:
again, there's probably a good reason but just want it explained here

Reply:
Good point, I can modify this to:

1. group charts by `chart.dataType`
2. submit different type of charts separately